tools/api_proto_breaking_change_detector/BUILD

```diff
@@ -1,28 +1,48 @@
-load("@rules_python//python:defs.bzl", "py_binary", "py_test")
+load("@rules_python//python:defs.bzl", "py_binary", "py_library", "py_test")
 
 licenses(["notice"])  # Apache 2
 
 py_binary(
     name = "proto_breaking_change_detector",
-    srcs = ["detector.py"],
+    srcs = [
+        "detector.py",
+    ],
     data = [
-        ":buf.lock",
-        ":buf.yaml",
         "@com_github_bufbuild_buf//:buf",
+        "@envoy_api_canonical//:proto_breaking_change_detector_buf_config",
     ],
     main = "detector.py",
     tags = ["manual"],
     deps = [
+        ":buf_utils",
+        ":proto_breaking_change_detector_errors",
         "//tools:run_command",
     ],
 )
 
+py_library(
+    name = "buf_utils",
+    srcs = [
+        "buf_utils.py",
+    ],
+    deps = [
+        ":proto_breaking_change_detector_errors",
+        "//tools/base:utils",
+    ],
+)
+
+py_library(
+    name = "proto_breaking_change_detector_errors",
```

Comment:
Convention is to have the target name match the file name where there's a single src file. Is there a reason the extra verbosity would be useful?